packages/pdf-generator/src/DownloadButton.js

```diff
@@ -5,77 +5,106 @@ import { Button } from '@patternfly/react-core';
 import PDFDocument from './components/PDFDocument';
 
 class DownloadButton extends React.Component {
-    state = {
-        asyncPages: []
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      asyncPages: []
     };
-    render() {
-        const {
-            fileName,
-            label,
-            isPreview,
-            asyncFunction,
-            buttonProps,
-            ...props
-        } = this.props;
+    this.updateAsyncPages = this.updateAsyncPages.bind(this);
```

Comment:
No need to use `bind` this repository is using autobinding from babel.

Reply:
removed